plugin/pkg/shoot/validator/admission.go

```
@@ -843,6 +843,49 @@ func (c *validationContext) validateAdmissionPlugins(a admission.Attributes, sec
 	return allErrs
 }
 
+// For backwards-compatibility, we want to validate the scaleDownUnneededTime and scaleDownDelayAfterAdd only for newly created Shoot clusters.
+// Performing the validation for all Shoots would prevent already existing Shoots with the wrong spec to be updated/deleted.
+func (c *validationContext) validateScaleDownConfiguration(a admission.Attributes) field.ErrorList {
+	var (
+		allErrs                field.ErrorList
+		stringRepresentation   string
+		path                   = field.NewPath("spec", "kubernetes", "clusterAutoscaler")
+		scaleDownUnneededTime  = 30 * time.Minute //default
+		scaleDownDelayAfterAdd = 1 * time.Hour    //default
+	)
+
+	if a.GetOperation() != admission.Create {
+		return nil
+	}
+
+	//in order to check in the case where the ScaleDownDelayAfterAdd is set generally, but the worker settings may set and overwrite ScaleDownUnneededTime
+	if c.shoot.Spec.Kubernetes.ClusterAutoscaler != nil {
+		if c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd != nil {
```

Isn't it better to throw an error if `c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd` are not set instead of using impicid defaults like
```
		scaleDownUnneededTime  = 30 * time.Minute //default
		scaleDownDelayAfterAdd = 1 * time.Hour    //default
```

---

Yes, it's incorrect to assume the defaulting. I think it's better not to throw an error if it's not defaulted, but maybe just stop validation altogether and `return nil` just like in the first check.